.github/ISSUE_TEMPLATE/request-for-comments.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Request for Comments
+about: Start a discussion around a feature, enhancement, or other change to DNN.Platform 
+---
+
+<!-- 
+  Please read contribution guideline first: https://github.com/dnnsoftware/Dnn.Platform/blob/development/CONTRIBUTING.md 
+  Any potential security issues should be sent to security@dnnsoftware.com, rather than posted on GitHub
+-->
+
+## Description of Problem
+Summarization of the solution requesting commentary.
+
+## Proposed Solution #1
```

Comment:
If we use the syntax #1 it will link to the first issue. I think we should use just the number 1 or add a space between them. Thoughts?

Reply:
Corrected